frontend/src/types/media.ts

```diff
@@ -35,50 +36,133 @@ export interface Media {
    * - handle empty titles
    */
   title: string
-  /** the raw name of the creative work, as returned by the API */
+
+  /** The raw name of the creative work, as returned by the API. */
   originalTitle: string
 
+  /** The text name, handle, or username of the author of the creative work. */
   creator?: string
+
+  /** A URL to the creator's profile on the provider or other personal webpage, depending on the provider. */
   creator_url?: string
 
+  /** A URL pointing to the actual media file on the provider. */
   url: string
+
+  /**
+   * A URL to the page where the media item is hosted on the foreign provider's site.
+   * This is often used to give credit to the original source or for users to find more information.
+   */
   foreign_landing_url: string
 
+  /** The code of the open copy-left license assigned to the work. */
   license: License
+
+  /** The version number of the Creative Commons license as a string, or an empty string. */
   license_version: LicenseVersion
+
+  /** A link to the landing page of the License, typically the deed or another informational page. */
   license_url?: string
+
+  /** The full text to properly attribute the work to the creator. */
   attribution: string
 
   frontendMediaType: SupportedMediaType
 
+  /** A long test describing the media, from the provider. */
   description?: string
 
-  category: string | null
+  /** A list of descriptive categories describing the work. */
```

Comment:
```suggestion
  /** The descriptive category describing the work. */
```

There's actually an Enum of possible categories in the API.